src/pages/workspaces/workspace/Tools.js

```diff
@@ -85,9 +86,8 @@ const ToolCard = pure(({ listView, name, namespace, config, onCopy, onDelete })
       })
     ])
   ])
-
   const repoLink = link({
-    href: sourceRepo === 'agora' ? `https://firecloud.dsde-dev.broadinstitute.org/#methods/${methodNamespace}/${methodName}/${methodVersion}`: `https://dockstore.org/workflows/${methodPath}`,
+    href: sourceRepo === 'agora' ? `${firecloudRoot}/#methods/${methodNamespace}/${methodName}/${methodVersion}`: `${dockstoreRoot}/workflows/${methodPath}`,
```

Comment:
I still think it would be a good idea to factor out this logic into a shared helper so we don't have to duplicate it in the other file.